src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.component.ts

```diff
@@ -41,26 +56,39 @@ export class FlowsTableComponent implements OnInit, OnChanges {
     @Input() public nodes: FlowSummaryDataFragment[];
     @Input() public filterByStatus: MaybeNull<FlowStatus>;
     @Input() public filterByInitiator: FilterByInitiatorEnum;
-    @Input() public searchByAccountName: string;
+    @Input() public searchByAccount: MaybeNull<Account>;
+    @Input() public searchByDataset: MaybeNull<Dataset>;
     @Input() public fetchStep: MaybeUndefined<FetchStep>;
     @Input() public transformData: TransformDescriptionTableData;
     @Input() public datasetBasics: DatasetBasicsFragment;
+    @Input() public accountView: boolean = false;
```

Comment:
Simply imagine yet another mode in future - admin should see all flows at once. This would destroy a lot of logic written based on `accountView` checks.